specification/containerservice/Fleet.Management/fleetmember.tsp

```diff
@@ -54,6 +64,11 @@ model FleetMemberProperties {
   @visibility("read")
   @doc("The status of the last operation.")
   provisioningState?: FleetMemberProvisioningState;
+
+  @added(Versions.v2025_03_01)
+  @doc("The labels to be persisted on the fleet member.")
```

Comment:
Please comment that they will be copied to the corresponding MemberCluster custom resource in hub cluster and whether there will be any transformation like adding a prefix to keys.

Reply:
Are we certain that the copy to the dataplane member will be implemented when this API rolls out?
if we document it in the API it must be. I think we could leave ourselves the room, and update the doc later.

Reply:
I agree with @serbrech on this point.

Reply:
sgtm